.gitignore

```diff
@@ -1 +1,13 @@
-node_modules
\ No newline at end of file
+node_modules
+dist/
+
+baseTrie.js
+checkpoint-interface.js
+index.js
+prioritizedTaskExecutor.js
+proof.js
+readStream.js
+secure-interface.js
+secure.js
+trieNode.js
+util.js
```

Comment:
I admit this solution is not super-pretty, but since there are also other ``.js`` files in the directory it's probably necessary to be so explicit here. I also double checked again, there seems to be no non-workaround solution in Node to set the import base path for modules (everyone correct me if I am wrong).